source/textInfos/__init__.py

```diff
@@ -551,6 +553,15 @@ def getMathMl(self, field):
 		"""
 		raise NotImplementedError
 
+	def _get_isOffscreen(self):
+		"""
+		Returns True if this textInfo is positioned outside its object's

```

Comment:
If the text is partially visible, what is returned?

Reply:
Depends on the underlying implementation.